fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/api/RunreportsApiResource.java

```diff
@@ -105,6 +109,10 @@ public Response runReport(@PathParam("reportName") @Parameter(description = "rep
         // Pass through isSelfServiceUserReport so that ReportingProcessService implementations can use it
         queryParams.putSingle(IS_SELF_SERVICE_USER_REPORT_PARAMETER, Boolean.toString(isSelfServiceUserReport));
 
+        if (parameterType) {
+            return datatableReportingProcessService.processRequest(reportName, queryParams);
+        }
+
```

Comment:
IMHO this should be a separate PR as it is not related to the topic / JIRA of this PR... can we split it out to a separate one with reference to a right JIRA? Otherwise it will be very confusing afterwards to work out what was changed where...

Reply:
I had toyed with the idea of splitting it, checked the code review guidelines and there was no mention of one PR commit one task one fix. Since it wasn't a code dump, I decided to fix it in this PR because it was blocking me from testing on a _related_ change on community-app..

I agree though with idea of one PR commit, one fix. 
- I will create a separate jira re Fix runreports
- Remove it from here
- And assign to you for review

Reply:
Fix in #1693 FINERACT-1345 
Please review. Thanks.
Somehow github isn't allowing me to select reviewer (no suggested reviewers) from PR top right.